### PR TITLE
Flashlist implementation in assets lists

### PR DESCRIPTION
**Pull request by banklesss**
Related to [YOMO-487](https://emurgo.atlassian.net/browse/YOMO-487)

[YOMO-487]: https://emurgo.atlassian.net/browse/YOMO-487?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Send/AssetSelectorScreen/AssetSelectorScreen.tsx [hunk 1]
```diff
@@ -66,8 +67,9 @@ export const AssetSelectorScreen = () => {
         )}
         bounces={false}
         contentContainerStyle={{paddingHorizontal: 16}}
-        keyExtractor={(tokenInfo) => tokenInfo.id}
+        keyExtractor={(_, index) => index + ''}
         testID="assetsList"
+        estimatedItemSize={78}
```
Comment (banklesss) [inline src/Send/AssetSelectorScreen/AssetSelectorScreen.tsx:72]: Approximated value https://shopify.github.io/flash-list/docs/usage/#estimateditemsize

File: src/TxHistory/AssetList/AssetList.tsx [hunk 2]
```diff
@@ -58,7 +59,8 @@ export const AssetList = (props: Props) => {
         )}
         ItemSeparatorComponent={() => <Spacer height={16} />}
         contentContainerStyle={{paddingTop: 16, paddingHorizontal: 16, paddingBottom: 8}}
-        keyExtractor={(tokenInfo) => tokenInfo.id}
+        keyExtractor={(_, index) => index + ''}
+        estimatedItemSize={78}
```
Comment (banklesss) [inline src/TxHistory/AssetList/AssetList.tsx:63]: Approximated value https://shopify.github.io/flash-list/docs/usage/#estimateditemsize

File: src/Send/AssetSelectorScreen/AssetSelectorScreen.tsx [hunk 3]
```diff
@@ -66,8 +67,9 @@ export const AssetSelectorScreen = () => {
         )}
         bounces={false}
         contentContainerStyle={{paddingHorizontal: 16}}
-        keyExtractor={(tokenInfo) => tokenInfo.id}
```
Comment (michalszorad) [inline src/Send/AssetSelectorScreen/AssetSelectorScreen.tsx:69]: Why is it not `tokenInfo.id` anymore?

Comment (banklesss) [reply src/Send/AssetSelectorScreen/AssetSelectorScreen.tsx:69]: Weird behaviour. Like for example repeated tokens. I looked at your changes for my implementation and it works!

**Review by michalszorad — APPROVED** [review]

File: src/Send/AssetSelectorScreen/AssetSelectorScreen.tsx (hunk 3 above)
Comment (banklesss) [reply src/Send/AssetSelectorScreen/AssetSelectorScreen.tsx:69]: Just change locally `FlatList` for `FlashList` and you will see

File: src/Send/AssetSelectorScreen/AssetSelectorScreen.tsx [hunk 4]
```diff
@@ -66,8 +67,9 @@ export const AssetSelectorScreen = () => {
...
+        keyExtractor={(_, index) => index + ''}
```
Comment (stackchain) [inline src/Send/AssetSelectorScreen/AssetSelectorScreen.tsx]: token id?

**Pull request merged**